CONTRIBUTING.rst

```diff
@@ -62,25 +62,21 @@ Ready to contribute? Here's how to set up `conda-devenv` for local development.
 
     $ git clone git@github.com:your_name_here/conda-devenv.git
 
-3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+3. Create a new conda environment for developing::
 
-    $ mkvirtualenv conda-devenv
-    $ cd conda-devenv/
-    $ python setup.py develop
+    $ conda create -n devenv --file requirements_dev.txt
+    $ conda develop %CD%\conda_devenv
```

Comment:
Does `conda develop` need an absolute path? Couldn't it simply be `cd conda-devenv` and `conda develop .`?